package.json

```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "@aws-amplify/analytics": "^3.1.7",
     "@aws-amplify/auth": "^3.2.4",
-    "@aws-amplify/ui-components": "^0.5.0",
+    "@aws-amplify/ui-components": "^0.5.2",
```

Comment:
What are your thoughts on using `latest`?  One problem is that `yarn.lock` won't respect it, so it'll require regenerating the lock file.

I feel that when docs are deployed, these should always be updated (such that daily deployments of the doc would always reflect `latest`).

Reply:
yea super good point